test/helpers/calculateImageHeight.js

```diff
@@ -0,0 +1,27 @@
+import chai from 'chai';
+import calculateImageHeight from 'helpers/calculateImageHeight';
+
+const { assert } = chai;
+
+describe('calculateImageHeight', () => {
+  it('returns the expected height', () => {
+    const actual = calculateImageHeight(1000, { w: 800, h: 600 });
+    const expected = 750;
+
+    assert.deepEqual(actual, expected);
```

Comment:
Since you're only comparing integers, I think you could use `assert.equal(actual, expected)`.